src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/TransportEquipmentGoodsRefStepDef.scala

```diff
@@ -68,14 +68,14 @@ class TransportEquipmentGoodsRefStepDef extends BaseStepDef {
       .submitPage()
   }
 
-  And("""^(?:I )?click the (.*) link on the You have added (.*) transport equipment page""") {
+  And("""^(?:I )?click the (.*) link on the You have added (.*) transport equipment page$""") {
     (sectionLink: String, numberOfTransportEquipment: String) =>
     AddAnotherTransportEquipmentPage
       .loadPage(numberOfTransportEquipment)
       .clickByPartialLinkText(sectionLink)
   }
 
-  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove transport equipment (.*) page""") {
+  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove transport equipment (.*) page$""") {
```

Comment:
Shall we keep those consistent?
`"""^(?:I )?click radio option (.*) on the 'Are you sure you want to remove transport equipment (.*)\?' page$"""`